Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ghcr.io/atlanhq/pyatlan-chainguard-base:8.3.0-3.11
```

Comment:
The same with pyatlan. Do we need a chainguard for pyatlan installed base image?

Reply:
I am still not convinced why we need to pre-install pyatlan in the base image? since it again will be installed as part of the apps